### DISCUSSION TITLE
Relation Design

### DISCUSSION THREAD

**Discussion by laurent512** [prisma/prisma#9121] [Q&A] [answered]
Hello Everyone,

Please apologize in advance for what might look as a basic question to you but I am a beginner in prisma & db design.

I have two models, Transaction and Reimbursement :  
- A **Transaction** record must be linked to  0 or 1 **Reimbursement** record
- A **Reimbursment** record  must be linked to 1 or many  **Transaction** record 

Would you be able to advice me on the best relationship model for that situation? 

Thank you so much in advance

**Comment by ryands17** [comment, marked as answer]
@laurent512 👋 

What you need is a 1-many relation as described [here](https://www.prisma.io/docs/concepts/components/prisma-schema/relations/one-to-many-relations/) where `Reimbursment` will be the `User` and `Transaction` will be the `Post` as shown in the example.

**Comment by ludralph** [comment]
Hi there,

To keep our discussions organized and focused on the most relevant topics, we’re reviewing and tidying up our backlog. As part of this process, we’re closing discussions that have already been marked as answered but remain open.

If this discussion still requires further input or clarification, feel free to reopen it or start a new one with updated details. Your contributions are invaluable to the community, and we’re here to help!

For more details about our priorities and vision for the future of Prisma ORM, check out our latest blog post: https://www.prisma.io/blog/prisma-orm-manifesto.

Thank you for your understanding and ongoing support of the Prisma community!
